python/mosaic/utils/kepler_magic.py

```diff
@@ -101,8 +101,25 @@ def mosaic_kepler(self, *args):
                 data = data.withColumn(
                     feature_name, lower(conv(col(feature_name), 10, 16))
```

Comment:
This conversion sounds something that should be inside the grid definition file. 
Maybe to refactor in a separate PR?